src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +17,13 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(x -> x.split(","))
+                .flatMap(Arrays::stream)
+                .mapToInt(Integer::parseInt)
+                .filter(x -> x % 2 == 0)
+                .min().orElseThrow(() ->
+                        new RuntimeException("Can't get min value from list: method_input_list"));
```

Comment:
```suggestion
                .min().orElseThrow(() ->
                        new RuntimeException("Can't get min value from list " + numbers));
```

what is method_input_list?